qiskit_nature/second_q/mappers/tapered_qubit_mapper.py

```diff
@@ -0,0 +1,197 @@
+
+        Return:
+            A ``TaperedQubitMapper`` with pre-built symmetry specifications.
+        """
+
+        qubit_op, _ = problem.second_q_ops()
+        mapped_op = mapper.map(qubit_op).primitive
+        z2_symmetries = Z2Symmetries.find_z2_symmetries(mapped_op)
+        tapering_values = problem.symmetry_sector_locator(z2_symmetries, mapper)
+        z2_symmetries.tapering_values = tapering_values
+        return TaperedQubitMapper(mapper, z2_symmetries)
+
+    def _map_clifford_single(self, second_q_op: SparseLabelOp) -> SparsePauliOp:
+        mapped_op = self.mapper.map(second_q_op).primitive
+        converted_op = self.z2symmetries.convert_clifford(mapped_op)
+        return converted_op
+
+    def _symmetry_reduce_clifford_single(
+        self, converted_op: SparsePauliOp, check_commutes: bool = True
+    ) -> None | SparsePauliOp:
+
+        if self.z2symmetries.is_empty() or self.z2symmetries.tapering_values is None:
+            return converted_op
+        elif check_commutes:
+            logger.debug("Checking operator commute with symmetries:")
+            converted_symmetries = self.z2symmetries._sq_paulis
+            commutes = TaperedQubitMapper._check_commutes(converted_symmetries, converted_op)
+
+            if not commutes:
+                return None
+
+        tapered_op = self.z2symmetries.taper_clifford(converted_op)
+        cast(SparsePauliOp, tapered_op)
+        return tapered_op
+
+    def _map_single(self, second_q_op: SparseLabelOp) -> PauliSumOp | None:
+        converted_op = self._map_clifford_single(second_q_op)
+        tapered_op = self._symmetry_reduce_clifford_single(converted_op)
+
+        returned_op = PauliSumOp(tapered_op) if isinstance(tapered_op, SparsePauliOp) else None
+        return returned_op
+
+    def map_clifford(
+        self,
+        second_q_ops: SparseLabelOp | ListOrDictType[SparseLabelOp],
+    ) -> PauliSumOp | ListOrDictType[PauliSumOp]:
+        """Maps a second quantized operator or a list, dict of second quantized operators based on
+        the current mapper.
+
+        Args:
+            second_q_ops: A second quantized operator, or list (resp. dict) thereof.
+
+        Returns:
+            A qubit operator in the form of a PauliSumOp, or list (resp. dict) thereof if a list
+            (resp. dict) of second quantized operators was supplied.
+        """
+        wrapped_type = type(second_q_ops)
+
+        if issubclass(wrapped_type, SparseLabelOp):
+            second_q_ops = [second_q_ops]
+
+        wrapped_second_q_ops: _ListOrDict[SparseLabelOp] = _ListOrDict(second_q_ops)
+
+        qubit_ops: _ListOrDict = _ListOrDict()
+        for name, second_q_op in iter(wrapped_second_q_ops):
+            qubit_ops[name] = PauliSumOp(self._map_clifford_single(second_q_op))
+
+        returned_ops: PauliSumOp | ListOrDictType[PauliSumOp]
+        returned_ops = qubit_ops.unwrap(wrapped_type)
+
+        return returned_ops
+
+    def symmetry_reduce_clifford(
```

Comment:
```suggestion
    def taper_clifford(
```
Just to be explicit and re-iterate on my previous proposal